packages/lib-classifier/src/components/Classifier/components/MetaTools/components/HidePreviousMarksButton/HidePreviousTranscriptionsButton.js

```diff
@@ -0,0 +1,114 @@
+import { SpacedText } from '@zooniverse/react-components'
+import counterpart from 'counterpart'
+import PropTypes from 'prop-types'
+import { Box, DropButton } from 'grommet'
+import { FormDown, FormView, FormViewHide, Hide } from 'grommet-icons'
+import React from 'react'
+import styled, { css, withTheme } from 'styled-components'
+import { MetaToolsButton } from '@zooniverse/react-components'
+
+import SHOWN_MARKS from '@helpers/shownMarks'
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+const StyledDrop = styled(DropButton)`
+  div {
+    margin-top: 10px;
+    padding: 0;
+  }
+
+  span {
+    ${props => css`line-height: ${props.theme.paragraph.small.height};`}
+  }
+`
+
+function HidePreviousTranscriptionsButton (props) {
+  const { disabled, onClick, shownMarks } = props
+  const [isOpen, setOpen] = React.useState(false)
+
+  const titles = {
+    ALL: counterpart('HidePreviousTranscriptionsButton.show'),
+    USER: counterpart('HidePreviousTranscriptionsButton.showUser'),
+    NONE: counterpart('HidePreviousTranscriptionsButton.hide')
+  }
+  const currentTitle = titles[shownMarks]
+
+  return (
+    <StyledDrop
+      a11yTitle={currentTitle}
+      dropContent={
+        <Box role='radiogroup'>
+          <MetaToolsButton
+            a11yTitle={counterpart('HidePreviousTranscriptionsButton.show')}
+            active={shownMarks === SHOWN_MARKS.ALL}
+            aria-checked={shownMarks === SHOWN_MARKS.ALL}
+            icon={<FormView />}
+            margin={{ top: '0' }}
+            onClick={() => {
+              onClick(SHOWN_MARKS.ALL)
+              setOpen(false)
+            }}
+            padding='0.5em'
```

Comment:
The `MetaToolsButton` is also a `plain` Grommet `Button`, so this isn't doing anything either. I may not have been clear about that. It'll have to be modified to take the `pad` prop and do something with it like apply styling via styled-components. This can be in a follow up PR?